book/fileParser.rb

```diff
@@ -0,0 +1,13 @@
+# This files is used to parse files into lines.
+# It could be switched to:
+#   1. more performant read method,
+#   2. enhanced with fringe case handling.
+class FileParser
```

Comment:
* `fileParser.rb` is not idiomatic file name for Ruby. It should be `file_parser.rb`
* This class is not a parser. It's rather a file loader
* You probably don't need this class at all if you use [`File.readlines`](https://apidock.com/ruby/IO/readlines) inherited from `IO`